codex-ui/src/vue/tabbar/Tab.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div :class="[$style.tab]">
+    <div
+      v-if="picture || icon"
+      :class="[$style['tab__left']]"
+    >
+      <template v-if="picture">
+        <img
+          :src="picture"
+          :class="[$style['tab__image']]"
+        />
+      </template>
+      <template v-else-if="icon">
+        <Icon :name="icon" />
+      </template>
+    </div>
+
+    <div :class="[$style['tab__center'], 'text-ui-base-medium']">
+      {{ name }}
+    </div>
+    <div
+      v-if="closable"
+      :class="[$style['tab__right']]"
+    >
+      <Icon :name="'Cross'" />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { TabState } from './Tab.types';
+import Icon from '../icon/Icon.vue';
+
+withDefaults(
+  defineProps<{
+    /**
+     * Name of the tab item
+     */
+    name: string;
+
+    /**
+     * If true we have cross icon on the right
+     */
+    closable?: boolean;
+
+    /**
+     * Current tab state
+     */
+    state: TabState;
```

Comment:
if there can only be 2 states, maybe it is easier to have just `isActive?: boolean` prop?